### ISSUE TITLE
Stopped working

### ISSUE THREAD

**Issue by nmaxcom** [biati-digital/alfred-calculate-anything#7]
AFAIK I didn't touch or change anything. No matters what do I write (even copying examples from the readme) it doesn't work:

![image](https://user-images.githubusercontent.com/8936090/84259286-8e1c3700-ab18-11ea-9469-fe2f1df4c653.png)

In Alfred everything seems okay, the workflow is there, enabled and so on. In fact, this works:

![image](https://user-images.githubusercontent.com/8936090/84259347-a2603400-ab18-11ea-8f84-9964b44d61e6.png)

I'm up to date. What can it be?

**Comment by nielsvanrijn** [comment]
+1 I started to have this issue too.
Deleted and reinstalled the workflow.
[Example here](https://streamable.com/y56fht)

**Comment by soyfeng** [comment]
Sadly same is happening on my machine – MacOS Catalina, Calculate Anything up to date. No changes made to any settings. 
![Screenshot 2020-06-23 at 8 52 46 PM](https://user-images.githubusercontent.com/67339540/85445474-bc2b5d80-b593-11ea-9c0e-b480b2acb07b.jpg)

**Comment by biati-digital** [comment]
Hi guys, I've added an issue template, please follow that template to provide the required information.

Also you can try removing the workflow cache: Go to /youruser/Library/Application Support/Alfred/Workflow Data/com.alfred.calculateanything/ and delete the folder cache.

**Comment by nielsvanrijn** [comment]
Thx! This fixed it in my case🎉

**Comment by biati-digital** [comment]
I'm glad it worked, if someone is kind enough to share their cache folder (when the workflow is not working) it will (hopefully) help me to find exactly what happened

**Comment by soyfeng** [comment]
It worked after I deleted the cache folder and removed the workflow, downloading the latest version again. Unfortunately, I didn't keep the old cache folder, but will share it if it happens again. Thanks very much for this excellent workflow 👍

**Comment by nmaxcom** [comment]
Didn't work for me.

I'm attaching the cache folder I had before deleting it.
[cache.zip](https://github.com/biati-digital/alfred-calculate-anything/files/4836017/cache.zip)

**Comment by mwhdc** [comment]
I believe the Alfred 4.1 update broke this workflow. The following error appears in debug logs when the workflow is invoked.

```
[22:25:54.841] Calculate Anything[Script Filter] Warning: unlink(/Users/mwh/Library/Application Support/Alfred/Workflow Data/com.alfred.calculateanything/remote-plist.plist): No such file or directory in /Users/mwh/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.A5A595BB-0581-4BB5-B21F-3D1323E673B7/alfred/updater.php on line 183
{"items": [{"title":"7.62cm","arg":"7.62","subtitle":"Action this item to copy this number to the clipboard","valid":true,"mods":{"cmd":{"valid":true,"arg":"7.62","subtitle":"Action this item to copy the amount with no format"},"alt":{"valid":true,"arg":"7.62cm","subtitle":"Action this item to copy this value with unit to the clipboard"}}}] }
[22:25:54.842] ERROR: Calculate Anything[Script Filter] JSON error: JSON text did not start with array or object and option to allow fragments not set. in JSON:
Warning: unlink(/Users/mwh/Library/Application Support/Alfred/Workflow Data/com.alfred.calculateanything/remote-plist.plist): No such file or directory in /Users/mwh/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.A5A595BB-0581-4BB5-B21F-3D1323E673B7/alfred/updater.php on line 183
{"items": [{"title":"7.62cm","arg":"7.62","subtitle":"Action this item to copy this number to the clipboard","valid":true,"mods":{"cmd":{"valid":true,"arg":"7.62","subtitle":"Action this item to copy the amount with no format"},"alt":{"valid":true,"arg":"7.62cm","subtitle":"Action this item to copy this value with unit to the clipboard"}}}] }
```

**Comment by biati-digital** [comment]
H guys, i'll publish an update this weekend.

**Comment by biati-digital** [comment]
A new update has been published 2.0.3, if you are not able to update directly from the shortcode please try to update it manually just download the latest version from the [releases page](https://github.com/biati-digital/alfred-calculate-anything/releases)

Please try it and let me know if it's working for you.

**Comment by mwhdc** [comment]
Working now! Thank you for publishing this and keeping it updated. Incredibly useful workflow.
